<?xml version="1.0" ?>
<launch>
  <node name="RiCTraffic" output="screen" pkg="ric_board" type="Start.py"/>
  <node name="laser_scanner" output="screen" pkg="hokuyo_node" type="hokuyo_node">
    <param name="port" value="/dev/Hokuyo"/>
    <param name="frame_id" value="Laser_link"/>
  </node>
  <node name="front_camera" output="screen" pkg="usb_cam" respawn="true" type="usb_cam_node">
    <param name="video_device" value="/dev/video0"/>
    <param name="camera_frame_id" value="Camera_link"/>
    <param name="pixel_format" value="yuyv"/>
    <param name="image_width" value="640"/>
    <param name="image_height" value="480"/>
  </node>
  <param command="$(find xacro)/xacro.py '$(find ric_description)/lizi/lizi_no_plugins.xacro'" name="robot_description"/>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher"/>
  <rosparam command="load" file="$(find ric_board)/config/lizi101.yaml"/>
</launch>
